NOnion/Utility/NetworkUtils.fs

```diff
@@ -1,14 +1,12 @@
 ﻿namespace NOnion.Utility
 
 open System
-open System.Runtime.ExceptionServices
 
-open FSharpx.Collections
 open Fsdk
 
 open NOnion
 
-module FSharpUtil =
+module NetworkUtils =
```

Comment:
Let's call this AsyncUtil instead, WithTimeout is not directly related to Network.

Reply:
Renamed to `AsyncUtil`.